scripts/common.py

```diff
@@ -70,9 +55,9 @@
 
 # Buf version.
 BUF_VERSION = '0.29.0'
-# Protoc is the compiler for protobuf files and the version must be same as
-# the version of protobuf library being used.
-PROTOC_VERSION = PROTOBUF_VERSION
+
+# Must match the version of protobuf in requirements_dev.in.
```

Comment:
How will you guarantee this -- perhaps a backend test? (That reads the file contents and does text matching?)

I can totally see the latter getting updated but not the other.

Reply:
done (added backend test)